forte/data/data_pack.py

```diff
@@ -867,25 +868,55 @@ def get_data(
         valid_context_ids: Set[int] = self._index.query_by_type_subtype(
             context_type_
         )
+
         if context_components:
             valid_component_id: Set[int] = set()
             for component in context_components:
                 valid_component_id |= self.get_ids_by_creator(component)
             valid_context_ids &= valid_component_id
 
+        def get_annotation_list(c_type):
+            """
+            Get an annotation list of a given context type.
+            """
+            if issubclass(c_type, Annotation):
+                return list(self.annotations)
+            elif issubclass(c_type, AudioAnnotation):
+                return list(self.audio_annotations)
+            else:
+                raise NotImplementedError(
+                    f"Context type is set to {c_type}"
+                    "but currently we only support"
+                    "{Annotation, AudioAnnotation}"
```

Comment:
Though this is correct, using `{}` here near a f-string may lead to confusion to others.